### DISCUSSION TITLE
Rredirected URL issue

### DISCUSSION THREAD

**Discussion by ShaunBits** [supabase/supabase#23812] [Questions]
Hi there.

I've been working on with Google Auth via selfhosted docker. The entire function works fine except I some how get "#" in my URL right after redirected to my app.

e.g. 
`http://127.0.0.1:5173` → `http://127.0.0.1:5173/#`

.env file
```
...
## General
# SITE_URL=http://localhost:3000
SITE_URL=http://127.0.0.1:5173
ADDITIONAL_REDIRECT_URLS=http://127.0.0.1:5173
JWT_EXPIRY=3600
DISABLE_SIGNUP=false
API_EXTERNAL_URL=http://localhost:8000
...
```

**Comment by J0ELh** [comment]
did you figure out what wasn't working?
